client/client.go

```diff
@@ -12,7 +12,20 @@ import (
 	"time"
 )
 
-var logger = log.ClientLogger
+var logger log.StdoutLogger
+
+// init instantiates and configures client logger with default log level.
+func init() {
+	clientLogger, err := log.NewStdoutLogger("client", log.INFO, log.FORMAT_SHORT)
+	if err != nil {
+		panic(err)
+	}
+	logger = *clientLogger
+}
+
+func DisableLogging() {
```

Comment:
Please remove this wrapper function `DisableLogging` and the `SetLogLevel` in L30. The logger should be configured upfront, so there is no need to bother our awesome `Client` with logging auxilary functions ;)

Reply:
In addition, you will ntoice that L15 now reads `var Logger log.Logger` instead of `var logger log.StdoutLogger`, so that test functions can replace the client's default `StdoutLogger` with, for instance, newly introduced `NullLogger`. For this purpose, client package exports its logger (notice `Logger` instead of our previous `logger`).